webapp/channels/src/components/actions_menu/actions_menu.tsx

```diff
@@ -387,12 +394,10 @@ export class ActionMenuClass extends React.PureComponent<Props, State> {
                 open={this.props.isMenuOpen}
                 onToggle={this.handleDropdownOpened}
             >
-                <OverlayTrigger
-                    className='hidden-xs'
-                    delayShow={500}
+                <WithTooltip
+                    id={`${this.props.location}_${this.props.post.id}_tooltip`}
+                    title={messages.tooltipTitle}
```

Comment:
You can directly pass <FormattedMessage..../> component here with the translation

Reply:
Updated